setup.cfg

```diff
@@ -11,7 +11,7 @@ combine_as_imports = True
 force_grid_wrap = 0
 include_trailing_comma = True
 known_first_party = httpx
-known_third_party = brotli,certifi,chardet,cryptography,h11,h2,hstspreload,nox,pytest,rfc3986,setuptools,trustme,uvicorn
+known_third_party = brotli,certifi,chardet,cryptography,h11,h2,hstspreload,nox,pytest,rfc3986,setuptools,tests,trustme,uvicorn
```

Comment:
I think this is a missing bit on `master` that appeared after #271 — I've seen you fix it in other places and I had to too in #214. Perhaps worth submitting a quick PR as it makes `nox -s lint` crash? I can take it if you'd like. :+1:

Reply:
Will do soon. Also mental note that `tests` should actually go in `known_first_party` instead (so that `isort` places it along with other sources instead of topmost like other external packages).